modules/jupyter-widget/src/widget.js

```diff
@@ -10,10 +10,10 @@ import {
   waitForElementToDisplay
```

Comment:
Minimal changes here–essentially cleaning up the file to conform to patterns elsewhere in the repo (e.g., using import instead of require)